runtime/src/bank.rs

```diff
@@ -1926,15 +1926,15 @@ impl Bank {
                 return;
             };
 
-        assert!(
-            self.epoch_schedule.get_slots_in_epoch(self.epoch)
-                > self.get_reward_total_num_blocks(status.calculated_epoch_stake_rewards.len())
```

Comment:
this calculation is wrong. This is Vec<Vec, where outer vec is # of partitions. The fn expects total # stake accounts. `calculated_epoch_stake_rewards` is being renamed in a different pr.